# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../euler/compile-fix.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Nur euler-1.61.0.orig/src/command.c euler-1.61.0/src/command.c
--- euler-1.61.0.orig/src/command.c	2005-10-24 03:38:24.000000000 +0300
+++ euler-1.61.0/src/command.c	2005-11-01 20:08:13.000000000 +0200
@@ -235,7 +235,7 @@
 
 extern int builtin_count;
 extern builtintyp builtin_list[];
-extern int command_count;
+static int command_count;
 extern commandtyp command_list[];
 
 static void do_list (void)
@@ -732,8 +732,6 @@
 }
 
 
-static int command_count;
-
 commandtyp command_list[] = {
 	{"quit",c_quit,do_quit},
 	{"hold",c_hold,ghold},
